src/git/Repository.ts

```diff
@@ -120,7 +120,7 @@ export class Repository {
     public fetch(): Promise<void> {
         return new Promise<void>((resolve, reject) => {
             Global.isVerbose() && console.log(`fetching repo ${this.repoName}`);
-            this.git.fetch((err) => {
+            this.git.fetch(['--all', '--tags'], (err) => {
```

Comment:
On the build pipeline this will cause the download of the whole repo - as we only have a shallow clone. For a CiCD pipeline we should not use this fetch at all (for local development this would be OK).